android/library/src/main/java/com/liferay/mobile/screens/webcontent/list/WebContentListScreenlet.java

```diff
@@ -64,7 +78,7 @@ protected View createScreenletView(Context context, AttributeSet attributes) {
 		TypedArray typedArray =
 			context.getTheme().obtainStyledAttributes(attributes, R.styleable.WebContentListScreenlet, 0, 0);
 
-		folderId = castToLongOrUseDefault(typedArray.getString(R.styleable.WebContentListScreenlet_folderId), 0);
+		folderId = castToLong(typedArray.getString(R.styleable.WebContentListScreenlet_folderId));
 
```

Comment:
The "castToLong" method has a default value and it's zero. So we can use it instead of "castToLongOrUseDefault".